pandas/util/__init__.py

```diff
@@ -1,3 +1,30 @@
 from pandas.util._decorators import Appender, Substitution, cache_readonly  # noqa
 
+from pandas import compat
 from pandas.core.util.hashing import hash_array, hash_pandas_object  # noqa
+
+# compatibility for import pandas; pandas.util.testing
```

Comment:
in the future pls put the issue numbers here with a comment to make removals easier.

when this code is read in the future, it will be very hard for anyone not part of the original conversation to grok what is happening here (easily).